elvee.c

```diff
@@ -303,6 +304,9 @@ int main(int argc, char **argv)
 
 #ifdef WINDOWS
 
+    for (int i = 0; i < argc; i++) {
+      argv[i] = argv_quote(argv[i]);
```

Comment:
Indentation nitpick:

```suggestion
        argv[i] = argv_quote(argv[i]);
```